### ISSUE TITLE
Hand picked product ordering

### ISSUE THREAD

**Issue by galapogos01** [woocommerce/woocommerce-gutenberg-products-block#4769]
## Is your feature request related to a problem? Please describe.
When choosing items for the Hand Picked Product block, they are sorted by the default Woocommerce sort order options (name, newness, etc).

## Describe the solution you'd like
It would be amazing if the Hand Picked Product block allowed the author to custom sort the order order of the products in the widget, ideally through something like drag and drop of the items in the selector.

## Describe alternatives you've considered
na

## Additional context
na

**Comment by Aljullu** [comment]
Thanks for opening this issue @galapogos01. While the Hand-picked Products block doesn't allow ordering products right inside the block, you can tweak the order products are shown as follows:

1. In the Hand-pick Products block, select the _Order products by: Menu order_ option.

<img src="https://user-images.githubusercontent.com/3616980/133965987-c57c4050-e756-4d30-a685-5e611c8f5567.png" width="293" alt="Order products by: Menu order option" />

2. Now, in wp-admin, go to _Products_.
3. There, switch to the _Sorting_ tab:

<img src="https://user-images.githubusercontent.com/3616980/133966061-ab5833e5-e462-430b-be06-a6bffa2780c8.png" width="352" alt="Sorting products" />

4. You will be able to reorder the products there using drag and drop. That order will be honored in the Hand-picked Products block you just added, as well as other parts of WooCommerce like the Shop page.

You can read more on how to do it from this page:

https://docs.woocommerce.com/document/managing-products/#sorting-products

Hope it helps! I will proceed closing this issue, but feel free to comment again if needed.

**Comment by galapogos01** [comment]
@Aljullu can you please re-open the ticket? The proposed solution changes the global product sort order everywhere, and is not a viable solution for an actual store with 4500 products.

Thanks,
Jason

**Comment by Aljullu** [comment]
Fair enough. I will keep this issue open, but to make expectations clear, we don't have plans to implement something like this directly in the block in the short-term.

**Comment by onthisearth** [comment]
@Aljullu Would love to see this implemented, too.

**Comment by galapogos01** [comment]
Please don't close this, I'm not the only person asking for it :)

**Comment by ijgeoffrey** [comment]
I definitely feel like this should be a feature. Why hand-pick products without hand-picking order? As it is I'm gonna have to try to find a workaround, and for reasons mentioned above, menu order is not a viable solution.

**Comment by katiebethbrown** [comment]
Adding a +1 as we ran into this on a Team 51 partner site. We were able to work around it with the menu order, but it would be nice to be able to set the order explicitly in the block without impacting the other areas of the site, similar to changing the order of images in a gallery.

**Comment by tommusrhodus** [comment]
@Aljullu I have added a PR that adds basic sorting abilities to this block #6666 to assist with the partner site mentioned by @katiebethbrown

**Comment by tommusrhodus** [comment]
Whilst my PR above would not work with the direction that WooCommerce Blocks is heading in, I have added my Proof of Concept to a seperate repo that can be used by anyone to allow sorting this block as a stop-gap until future updates to the core plugin:

https://github.com/Automattic/handpicked-products-order

Note: This stop-gap is exactly that, it just fills a need for now, this plugin will become obsolete in future WooCommerce Blocks updates.
